x-pack/platform/plugins/shared/cases/server/saved_object_types/import_export/export.test.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { httpServerMock, loggingSystemMock, coreMock } from '@kbn/core/server/mocks';
+import type { SavedObjectsExportTransformContext } from '@kbn/core/server';
+import { handleExport } from './export';
+import { mockCases } from '../../mocks';
+import type { CaseSavedObjectTransformed } from '../../common/types/case';
+
+jest.mock('./utils', () => {
+  return {
+    getAttachmentsAndUserActionsForCases: jest.fn().mockResolvedValue([]),
+  };
+});
+
+describe('case export', () => {
```

Comment:
What is the intended user experience when re-importing? Might be a bit jarring to lose all case numbers 🤷🏻

Reply:
There's no way for us to trust the imported case numbers. If these would get imported into another space, they could have conflicts.